src/components/molecules/ResourceFilter/ResourceFilter.tsx

```diff
@@ -187,12 +179,32 @@ const ResourceFilter = () => {
 
   return (
     <S.Container>
-      <S.Title>
-        <S.TitleLabel>Filter resources by:</S.TitleLabel>
-        <S.TitleButton type="link" onClick={resetFilters} disabled={areFiltersDisabled}>
-          Reset all
-        </S.TitleButton>
-      </S.Title>
+      <S.TitleContainer>
```

Comment:
Nitpick but the UI does not look good if you put navigator on minimum width.

I also find 'Load preset' and 'Save preset' a bit verbose but that's a bit personal.

<img width="371" alt="Screenshot 2022-05-25 at 12 33 40" src="https://user-images.githubusercontent.com/7761005/170242890-b2697b14-6d88-46a8-ae6c-72972ad5438e.png">

Reply:
We could show "Load" and "Save" when there's not enough space